wordpress.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+	extends: ['plugin:@wordpress/eslint-plugin/custom', './react'],
```

Comment:
@nicholasio I had a chance to test this today. Looks like this isn't pulling the correct rules. Many things that should be an error weren't treated as errors.

I used `plugin:@wordpress/eslint-plugin/recommended-with-formatting` as suggested here: https://github.com/WordPress/gutenberg/tree/master/packages/eslint-plugin . This pulls the correct rules, and doesn't use Prettier, so we can avoid the conflict you were referring to.

I believe we can also avoid extending `react`.

Reply:
If you pull `plugin:@wordpress/eslint-plugin/recommended-with-formatting` and `react` do you still see errors and warnings you would expect from the wp config? It might be good to keep `react` just to make sure there's a common denominator between our react config and the wp one.

Reply:
@nicholasio Looks like `plugin:@wordpress/eslint-plugin/recommended-with-formatting` already extends react? https://github.com/WordPress/gutenberg/blob/master/packages/eslint-plugin/configs/recommended-with-formatting.js#L6

Reply:
@nicholasio I can't test it properly because I keep getting an error: `ESLint couldn't find the plugin "@10up/eslint-plugin-eslint-config"`. Any idea how I can fix that?

Reply:
@nicholasio Apologies for the barrage of comments, I was able to test this now - and to confirm - **yes there is a conflict** in rules between `plugin:@wordpress/eslint-plugin/recommended-with-formatting` and `react`. Especially to do with spaces.

So I think both can't be used together, unfortunately 😞

This was the winning config for me:

```js
module.exports = {
	extends: ['plugin:@wordpress/eslint-plugin/recommended-with-formatting'],
	globals: {
		window: true,
		document: true,
		'wp': true,
		'lodash': true
	},
	rules: {
		'@wordpress/dependency-group': 0,
	},
};
```

Reply:
The problem with this is that we would have two completely different eslint configs within one package. I think we can extend `./react` and `plugin:@wordpress/eslint-plugin/recommended-with-formatting`, then `eslint-config-prettier` will disable all formatting rules and `./react` will override any conflicting rules to match ours. We would still get all the specific WP stuff.

See my latest commit.

Reply:
We're looking for consistency here. At the end of the day, we want all 10up engineers to write consistent JavaScript. Our open-source plugins are still 10up projects so I don't see any issue imposing our own rules, which are based on industry standards.

I'm in touch with our open-source team and the feedback so far has been great! I already got feedback about bugs and issues being caught by this new eslint config.

With that said the WordPress eslint rules can still be used directly, if necessary, although our goal is to use this one as the de-facto 10up standard.